### ISSUE TITLE
alternate thumbnail

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#324]
following on from #323 - looking at en.wikipedia.org/wiki/Help:Pictures and the section 'Linking without displaying'
xowa
![alt_thumb_bad](https://user-images.githubusercontent.com/861631/50962785-e7ff7400-14c2-11e9-98bc-ae7499fa795f.jpg)
enwiki
![alt_thumb_good](https://user-images.githubusercontent.com/861631/50962803-f3529f80-14c2-11e9-8143-76d2faf784fe.jpg)
xowa is not taking the `thumb=Anime stub.png` into account

There is currently no code to handle this, I had a look but got confused (bear of little brain)
Any change here I suspect would seriously impact hzip - but so what
As the wiki itself calls it a rare case, this is truly an edge case

**Comment by gnosygnu** [comment]
Yeah, I'm hoping this is an edge case. I think it requires a minor change to [the File parser](https://github.com/gnosygnu/xowa/blob/166963d7675354d744828a0b9e18fef78e70ff3d/400_xowa/src/gplx/xowa/parsers/lnkis/Xop_lnki_wkr.java) but I don't have the head now either to figure it out. I'll keep this issue open until I get around to it or until it shows up more frequently.

Thanks.
